rs/ic_os/dev_test_tools/launch-single-vm/src/main.rs

```diff
@@ -97,14 +96,11 @@ fn main() {
 
     // Adjust VM configuration for nested setup
     let (image_location, vm_type, vm_size) = if args.nested {
-        // Create an empty image, this will be used as the "main" drive
-        let empty_image_name = "empty.img.tar.zst";
-        let tmp_dir = tempfile::tempdir().unwrap();
-        let empty_image = build_empty_image(tmp_dir.path(), empty_image_name).unwrap();
+        let empty_disk_img_path =
+            std::env::var("EMPTY_DISK_IMG_PATH").expect("EMPTY_DISK_IMG_PATH not set");
         let empty_image_id = farm
-            .upload_file(&group_name, empty_image, empty_image_name)
+            .upload_file(&group_name, empty_disk_img_path, "empty-disk-img.tar.zst")
```

Comment:
We are still uploading the image here from `EMPTY_DISK_IMG_PATH`. Could we move this to `launch-remote-vm.sh` as we do [here](https://github.com/dfinity/ic/blob/621c059cc54cb72c7b023cab27c7e95e3b58ef03/ic-os/dev-tools/launch-remote-vm.sh#L7), to break this too out of the Rust code?

Reply:
Ah I think I just missed the cutoff. No reason this can't wait for a followup though 😁.

Reply:
Yes, I think we can. Let me code that up. Thanks for the suggestion!

Reply:
@Bownairo I pushed https://github.com/dfinity/ic/pull/4407/commits/70c43211560bec7f973f2efb356f5a13ac8c3b2e to implement your suggestion and tested it with a successful:
```
bazel run ic-os/setupos/envs/dev:launch-remote-vm
```

Reply:
Looks great! Thank you 🙌.